components/colorwheel/stories/colorwheel.stories.js

```diff
@@ -76,7 +110,9 @@ Disabled.parameters = {
  *
  * The `.spectrum-colorwheel-colorarea-container-size` is hard coded to position the color area within the color wheel using `.spectrum-color-wheel-color-area-margin`. Implementations using JS can calculate the container size with `Math.sqrt(2 * R * R)`, where `R` is the inner radius as calculated for the clip paths.
  *
- * `.spectrum-colorwheel-path`, `.spectrum-colorwheel-path-borders` and `.spectrum-colorwheel-colorarea-container` are hard coded in CSS, and include token values as custom CSS variables so they can be accessed with JS. To use and calculate these values, implementations should consider:
+ * Implementations should factor in the value of --spectrum-color-wheel-color-area-margin, as illustrated in the token specs, when calculating the size of the color area relative to the color wheel that contains it.
```

Comment:
This doesn't block an approval or anything, but we could add the backticks around this property (although they aren't rendering right now, not because of something you did!).

```suggestion
* Implementations should factor in the value of `--spectrum-color-wheel-color-area-margin`, as illustrated in the token specs, when calculating the size of the color area relative to the color wheel that contains it.
```